eternalcombat-api/src/main/java/com/eternalcode/combat/fight/controller/FightMessageController.java

```diff
@@ -56,6 +59,13 @@ void onUnTag(FightTagEvent event) {
 
         this.announcer.sendMessage(player, this.config.messages.playerUntagged);
         this.bossBarService.hide(event.getPlayer());
-    }
 
+        if (event.getCause() == CauseOfUnTag.LOGOUT) {
+            Formatter formatter = new Formatter()
+                    .register("{PLAYER}", player.getName());
+
+            String format = formatter.format(this.config.messages.playerLoggedOutDuringCombat);
+            this.announcer.broadcast(format);
+        }
```

Comment:
to można przenieść do logout controller, jest bardziej powiązane z samym logoutem